src/controllers/user-controller.js

```diff
@@ -0,0 +1,70 @@
+const userService = require("../service/user-service");
+
+class UserController {
+  async registration(req, res, next) {
+    try {
+      const { login, password } = req.body;
+
+      const userData = await userService.registration(login, password);
+
+      res.cookie("refreshToken", userData.refreshToken, {
```

Comment:
```
{
  maxAge: 30 * 24 * 60 * 60 * 1000,
  httpOnly: true,
}
```
this object is repeated, so move it higher into a constant

and this is 30 * 24 * 60 * 60 * 1000,
no one should count himself, so write in the comment what time it is here